### ISSUE TITLE
Form rendering Bug

### ISSUE THREAD

**Issue by RisteDimitrievski** [symfony/symfony#27509]
**Symfony version(s) affected**: x.y.z
4.1*
**Description**  

Error trying to render a form, see a link bellow (screenshot)
http://prntscr.com/jrduia
http://prntscr.com/jrdvb3
**How to reproduce**  

**Possible Solution**  

**Additional context**  

Controller:
http://prntscr.com/jrduwc
Form type: (App/Form/LostpasswordType)
http://prntscr.com/jrdv6i

Entity:  http://prntscr.com/jrdw0l
Error message:
http://prntscr.com/jrdvb3

**Comment by xabbuh** [comment]
Can you please create a small example application that allows to reproduce your issue and that we can check out to see what is going wrong?

**Comment by RisteDimitrievski** [comment]
I can give you the app,  via email. No need to create a small applications because im working on big project which i need fast to finish.

I'm frustrated because all the pages are in the same way coded, and only on this page is showing the bug. 
Before in 4.0 i wasn't having similar problem.

**Comment by xabbuh** [comment]
Well, without a reproducible example we won't be able to investigate the bug. Having a small example makes it easier to find out where the problem is and minimises the probability that the observed behaviour is not a bug but a misconfiugration/misuse in your project. Also publishing the example on, for example, GitHub allows for others contributors to look into the issue too.

**Comment by RisteDimitrievski** [comment]
Look xabbuh, i've installed the project via  
'
composer create-project symfony/website-skeleton
.
And if it was a misconfiguration, it will be on all pages not in just that route.

If i every private project get alot of bugs causing twig or symfony, and publishing it in public then i just leaving the symfony as Framework.

I understand you clearly, but it is not my fault because the Framework is alot bugged in 4.1 than in 4.0.

Here is the other pages: http://prntscr.com/jrjl3s

Controller for login:  http://prntscr.com/jrjlmg
Entity for login:  http://prntscr.com/jrjm8j
Form for login (App\Form\LoginType):  http://prntscr.com/jrjmmt

Now see, is this a missconfiguration or a bug. :).
If it was missconfiguration i will see that error on login, on registration too.

**Comment by javiereguiluz** [comment]
@RisteDimitrievski I see that you add the `submit` field in your form builder ... but then the page shows an error saying that `form.submit` is not accessible.

I think is safe to assume that it's not a general Symfony Form bug because we would have receive a ton of error reports from lots of users. I have updated myself some Symfony apps to 4.1 and they have forms and add buttons in the form and they work. So, this may be an edge-case bug ... and those are tricky to debug. That's why @xabbuh asked you a reproducer (as explained in https://symfony.com/doc/current/contributing/code/reproducer.html) so we can investigate and fix this quickly. Without a reproducer it will take us more time. We're sorry!

**Comment by RisteDimitrievski** [comment]
http://www.mediafire.com/file/q6uho537uiica3u/gamepanel.zip
Download it and investigate the bug.

**Comment by xabbuh** [comment]
The form builder of your `LostpasswordType` class is never used as the method is named `builder()` there. If you rename it to `buildForm()`, everything will work as expected. I am closing here as there is nothing to fix.

**Comment by javiereguiluz** [comment]
@xabbuh thanks for taking a look into this. Please, let me reopen this issue so @RisteDimitrievski can have some time to test the change you mentioned and he can confirm that the bug was that. Thanks!

**Comment by RisteDimitrievski** [comment]
Yes the bug was that, i was hurrying in the coding and make some mistake in the method as Mr @xabbuh  said.

Thanks for the support, i've patched the bug and now everything is fine.

**Comment by javiereguiluz** [comment]
@RisteDimitrievski thanks for verifying this and thanks for providing the bug reproducer. Thanks to @xabbuh too for debugging this. Let's close as fixed then.
